Runtime/Native/Android/NativeClient.cs

```diff
@@ -257,10 +255,9 @@ private void HandleNativeCrashes(IDictionary<string, string> backtraceAttributes
                 Directory.CreateDirectory(databasePath);
             }
 
-            var apiLevelString = backtraceAttributes["device.sdk"];
-            int apiLevel;
-            if (apiLevelString == null || !int.TryParse(apiLevelString, out apiLevel))
+            if (!backtraceAttributes.TryGetValue("device.sdk", out var apiLevelString) || !int.TryParse(apiLevelString, out var apiLevel))
```

Comment:
even in the newest version of unity you can still target older .net sdks - we should stick to the previous api